boards/st/sensortile_box_pro/doc/index.rst

```diff
@@ -214,6 +217,13 @@ The final application may use it to declare SensorTile.box PRO device as belongi
 certain standard or vendor class, e.g. a CDC, a mass storage or a composite device with both
 functions.
 
+BlueNRG-LP chip
+===============
+
+The board is equipped with a ST Microelectronics `BlueNRG-LP`_ chip. Before using this
```

Comment:
```suggestion
The board is equipped with an ST Microelectronics `BlueNRG-LP`_ chip. Before using this
```

Reply:
Sorry, I missed it. I'll review your inputs carefully tomorrow.

Reply:
"STMicroelectronics" is attached, no!?

Reply:
> "STMicroelectronics" is attached, no!?

Oh yes. Thanks for noticing it. I'll correct it